lib/migration_investigation.rb

```diff
@@ -6,7 +6,7 @@ def pending_migrations?(stage, new_sha, options={})
       old_sha = latest_stage_sha stage
       return true if old_sha.nil?
 
-      change_count(old_sha, new_sha, build_interesting_paths(options)) != 0
+      change_count(old_sha, new_sha, build_interesting_paths(options.fetch(:additional_paths, [])), options[:skip_pattern]) != 0
```

Comment:
@nherzing Thoughts on calling that option `skip_filepath_pattern` just to make it clear that it's applying the pattern to the filepath?